frost-core/src/frost.rs

```diff
@@ -255,43 +279,41 @@ pub struct GroupCommitment<C: Ciphersuite>(pub(super) Element<C>);
 //     }
 // }
 
-impl<C> TryFrom<&SigningPackage<C>> for GroupCommitment<C>
+/// Generates the group commitment which is published as part of the joint
+/// Schnorr signature.
+///
+/// Implements [`compute_group_commitment`] from the spec.
+///
+/// [`compute_group_commitment`]: https://www.ietf.org/archive/id/draft-irtf-cfrg-frost-10.html#section-4.5
+#[cfg_attr(feature = "internals", visibility::make(pub))]
+fn compute_group_commitment<C>(
```

Comment:
Can we also call this function from the `TryFrom` impl for the default (`internals` feature not enabled) case?

Reply:
We can, but it wouldn't be used. `compute_group_commitment` receives a previously-computed `binding_factor_list` (which _is_ previously computed everywhere it's called) so there is no motivation to actually call the TryFrom (which would recompute `binding_factor_list`). We could make a `TryFrom` from a tuple, but I think that's usually non-intuitive...